Fix rendering of std/var docs

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #91761
* __->__ #91730

Due to the indentation, "versionchanged" is being rendered as if it was an
argument.

peterbell10 committed:
Fix rendering of std/var docs

Due to the indentation, "versionchanged" is being rendered as if it was an
argument.

[ghstack-poisoned]

albanD reviewed (approved):
Looks good! Thanks for the update.

lezcano commented on torch/_torch_docs.py:
```diff
@@ -10701,11 +10701,12 @@ def merge_dicts(*dicts):
-        ``True`` corresponding to ``correction=1`` and ``False`` being ``correction=0``.
+        Defaults to `Bessel's correction`_, ``correction=1``.
 
+        .. versionchanged:: 1.14
```
Shouldn't this be 2.0?

albanD replied:
Yes it should :p

peterbell10 committed:
Update version numbering to 2.0 on "Fix rendering of std/var docs"


Due to the indentation, "versionchanged" is being rendered as if it was an
argument.

[ghstack-poisoned]
